src/senaite/jsonapi/api.py

```diff
@@ -271,6 +271,12 @@ def get_info(brain_or_object, endpoint=None, complete=False):
             return {}
         complete = True
 
+    # When querying uid catalog we have to be sure that we skip the objects
+    # used to relate two or more objects
+    if api.is_relationship_object(brain_or_object):
+        logger.warn("Skipping relationship object {}".format(brain_or_object))
+        return {}
+    
```

Comment:
Maybe a better approach to check here in general for:

```
if not api.is_object(brain_or_object):
    ...
```
and return then directly `{}`

Reply:
I would then probably also remove `api.is_relationship_object`, because I think it is not needed.

Reply:
@ramonski The problem here is that this _relationship_ objects are actually brains that pass the `is_object` test. From a pdb session:
```python
(Pdb) brain_or_object
<Products.Archetypes.UIDCatalog.plugbrains object at 0x7ff598999a60>
(Pdb) brain_or_object.getPath()
'bika_setup/bika_departments/department-4/at_references/f3d2b116bb214d20a24fa0bc20abe6fe'
(Pdb) api.is_object(brain_or_object)
True
```

Reply:
Thanks Juan, I think we can leave it like this. However, I would probably move the `is_relationship_object` into the domain of the `jsonapi.api`, because it is probably not needed somewhere else